creator-node/src/snapbackSM/peerSetManager.js

```diff
@@ -20,7 +21,7 @@ const MAX_NUMBER_SECONDS_PRIMARY_REMAINS_UNHEALTHY = config.get(
   'maxNumberSecondsPrimaryRemainsUnhealthy'
 )
 
-const DEFAULT_AXIOS_TIMEOUT_MS = 5000 // 5s
+const DEFAULT_AXIOS_TIMEOUT_MS = 10_000 // 10s
```

Comment:
does the `_` work here?

Reply:
Yup! it's probably the great programming UX feature ever added

Reply:
woah, TIL lol this is great